Powershell scripts/VM Scanners/onboardingTemplate.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+	"_generator": {
+	  "name": "bicep",
+	  "version": "0.5.6.12127",
+	  "templateHash": "2227781763411200690"
+	}
+  },
+  "parameters": {
+	"principalId": {
+	  "type": "string",
+	  "metadata": {
+		"description": "The principal to assign the role to"
+	  }
+	},
+	"targetRgName": {
+	  "type": "string",
+	  "metadata": {
+		"description": "The resource group in which the VmScanners resource will be created"
+	  }
+	},
+	"roleNameGuid": {
+	  "type": "string",
+	  "defaultValue": "[newGuid()]",
+	  "metadata": {
+		"description": "A new GUID used to identify the role assignment"
+	  }
+	},
+	"actions": {
+	  "type": "array",
+	  "defaultValue": [
+		"Microsoft.Compute/disks/read",
+		"Microsoft.Compute/disks/beginGetAccess/action",
+		"Microsoft.Compute/virtualMachines/instanceView/read",
+		"Microsoft.Compute/virtualMachines/read",
+		"Microsoft.Compute/virtualMachineScaleSets/instanceView/read",
+		"Microsoft.Compute/virtualMachineScaleSets/read",
+		"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/read",
+		"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/instanceView/read"
+	  ],
+	  "metadata": {
+		"description": "Array of actions for the roleDefinition"
+	  }
+	},
+	"notActions": {
+	  "type": "array",
+	  "defaultValue": [],
+	  "metadata": {
+		"description": "Array of notActions for the roleDefinition"
+	  }
+	},
+	"roleName": {
+	  "type": "string",
+	  "defaultValue": "Custom Role - VM Scanners User",
+	  "metadata": {
+		"description": "Friendly name of the role definition"
+	  }
+	},
+	"roleDescription": {
```

Comment:
roleDescription shouldnt be a parameter, set description directly with this value (line 86).
Also change the description to 'Used by Microsoft Defender for Cloud to scan VMs'